internal/uidriver/glfw/ui.go

```diff
@@ -514,6 +540,21 @@ func (u *UserInterface) SetCursorVisible(visible bool) {
 	})
 }
 
+func (u *UserInterface) SetCursorCaptured(captured bool) {
+	if !u.isRunning() {
+		u.setInitCursorCaptured(captured)
+		return
+	}
+	_ = u.t.Call(func() error {
+		c := glfw.CursorNormal
```

Comment:
Hmm, I'm not sure setting `normal` is fine when the cursor is hidden. If cursor visibility and capturing state is orthogonal, we would need to keep the states in boolean values and switch the GLFW state correctly. Wdyt?

Reply:
I was thinking that as well. A combined API for all cursor states would be better, but I didn't want to cause any breaking changes. 

I think I could just add a cursor state, so toggling cursor capture would revert to the previous state.

Let me think of the best way to do this and I'll post an update.

Reply:
I agree we should not break the compatibility. Another idea is to deprecate `Is/SetCursorVisibile` and add a new function to take enum values. I'm fine with both (adding the new functions as you did with some tricks, or adding a brand new function).

Thank you for considering!

Reply:
The cursor visibility API is now deprecated